app/templates/components/forms/orders/guest-order-form.hbs

```diff
@@ -5,23 +5,28 @@
     </div>
   {{/if}}
   <h4 class="ui header">{{t 'Please enter your email address to continue.'}}</h4>
-  <div class="field">
-    {{input type='text' name="email" value=email  disabled=userExists placeholder=(t 'Email Address')}}
+  <div class="field" style="
+    margin-bottom: 0px;">
+    {{input type='text' name="email" value=email  enabled=userExists placeholder=(t 'Email Address')}}
   </div>
   {{#if userExists}}
-    <div class="ui text muted">{{t 'You have previously registered with this email address.'}}</div>
+    <div class="ui pointing label" style="margin-bottom: 12px;">{{t 'You have previously registered with this email address.'}}</div>
     <div class="field">
       <label class="required" for="password">{{t 'Please enter your password to continue.'}}</label>
-      {{input type='password' name="password" value=password}}
+      {{input type='password' name="password" value=password placeholder=(t 'Password')}}
     </div>
     <div class="ui center aligned segment basic">
       <a href="{{href-to 'reset-password'}}" class="text muted"> {{t 'Forgot your password ?'}}</a>
     </div>
     <button type="submit" class="ui blue button">
       {{t 'Sign In'}}
     </button>
+    <button type="submit" class="ui blue button" value="Refresh Page" onClick="window.location.reload()">
```

Comment:
strictly no onclicks, no window.reload.
This logic is redundant if idea of clearing a field is fulfilled by reloading the entire app.

Reply:
@CosmicCoder96 We not only have to clear a field but to we have to open the previous pop-up which only asks for email ?